src/services.js

```diff
@@ -225,15 +225,28 @@ export class Services {
   }
 
   /**
-   * Returns service implemented in service/history-impl.
+   * Returns a service to register callbacks we wish to execute when an
+   * amp-form is submitted. This is the sync version used by amp-form only, all
```

Comment:
since it's by amp-form only, should it stay in amp-form?

Reply:
It is still a doc level service used by potentially many amp-forms, I though serviced was the preferred method for that?